examples/kdl-schema.kdl

```diff
@@ -20,7 +20,7 @@ document {
         max 1
         children id="node-children" {
             node "node-names" id="node-names-node" description="Validations to apply specifically to arbitrary node names" {
-                children ref="#validations"
+                children ref=r"[id="validations"]"
```

Comment:
I think this should be `r#"[id="validations"]"#` (same with the other lines).

Reply:
omg you're right. facepalm. And I thought I was being so clever lol